maven-artifact/src/test/java/org/apache/maven/artifact/versioning/ComparableVersionTest.java

```diff
@@ -383,4 +390,15 @@ void testMng7644() {
             checkVersionsEqual("2.0." + x, "2.0.0." + x); // previously ordered, now equals
         }
     }
+
+    @Test
+    public void testMng7714() {
+        String f = "1.0.final-redhat";
+        String sp1 = "1.0-sp1-redhat";
+        String sp2 = "1.0-sp-1-redhat";
+        String sp3 = "1.0-sp.1-redhat";
+        checkVersionsOrder(f, sp1);
```

Comment:
This is unclear since there's nothing in the method call to say which is expected to be greater. If these can be tested directly, it would be easier to follow. That is, show the comparisons between f and sp1, etc.

Reply:
I think it's possible to restandardize and redefine the approach to these tests.

Reply:
only new tests please. Do not change existing tests unless the behavior has changed.